Makefile

```diff
@@ -22,7 +22,7 @@ TEST_E2E_TIMEOUT ?= 20m
 ifeq ($(RELEASE_BUILD),true)
 	BUILTIN_CLOUD_PROVIDERS ?= aws azure ibmcloud vsphere
 else
-	BUILTIN_CLOUD_PROVIDERS ?= aws azure ibmcloud vsphere libvirt
+	BUILTIN_CLOUD_PROVIDERS ?= aws azure ibmcloud ibmcloud_powervs vsphere libvirt
```

Comment:
One additional comment.

Currently, there is no documentation about how to enable PowerVS in cloud-api-adaptor yet. How about adding `ibmcloud_powervs` in builtin provider list, when the documentation is ready?

Reply:
I second that. Let's add it to builtin list after documentation and may be few test CI runs

Reply:
Also required will be the basic podvm image creation steps for ppc64le.

Reply:
Sure, makes sense. I will submit a follow-up PR regarding the documentation and podvm image for PowerVS.